test/1_stdlib/Runtime.swift

```diff
@@ -1364,14 +1364,6 @@ Reflection.test("MirrorMirror") {
   expectEqual(0, mirrorMirror.children.count)
 }
 
-Reflection.test("OpaquePointer/null") {
-  // Don't crash on null pointers. rdar://problem/19708338
-  var sequence: OpaquePointer = nil
-  var mirror = Mirror(reflecting: sequence)
-  var child = mirror.children.first!
-  expectEqual("(Opaque Value)", "\(child.1)")
-}
```

Comment:
I couldn't figure out a way to write it that wasn't vacuously true or immediately undefined behavior.